src/mdx.ts

```diff
@@ -96,8 +96,8 @@ async function getDocsFrontMatterUncached(): Promise<FrontMatter[]> {
   return frontMatter;
 }
 
-export function getAllFilesFrontMatter(folder: string = 'docs') {
-  const docsPath = path.join(root, folder);
```

Comment:
```
function getAllFilesFrontMatter(folder: 'docs' | 'develop') {
	let docsPath;
	if (folder === 'docs') {
	  docsPath = path.join(root, 'docs');
	} else {
	  docsPath = path.join(root, 'develop');
	}
```

@a-hariti If we need to parameterize this function in the future we can do the above so that NFT continues to be able to analyze what is required for this file.